kwc-share-cover.html

```diff
@@ -28,6 +29,12 @@
             }
             .share-cover-image {
                 @apply --kwc-share-cover-image;
+                flex: 1;
+                width: 100%;
+                height: 100%;
+                --iron-image-placeholder: {
+                    background: var(--kwc-share-cover-placeholder, #939ed5);
```

Comment:
Set this colour to something like lightgrey by default

Reply:
👍